wearos_app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.example.homeassistantwearos">
```

Comment:
I am not sure if we want the package to remain as `com.example` maybe it should follow the main application?  I don't own a wear OS device so can't comment on anything else, this was all that stood out to me :)

Not sure what the actual format should be either, I believe changing it later once its installed on a device may cause issues though :/

Reply:
Good catch! I thought I already updated this but apparently not. I was thinking to keep it in line with the android package: io.homeassistant.companion.wearos

Reply:
I believe they need to be the same as well.

Reply:
You're right: https://developer.android.com/training/wearables/apps/packaging